libs/wire-api/src/Wire/API/Routes/Public/Brig.hs

```diff
@@ -384,6 +384,18 @@ type AccountAPI =
         :> ReqBody '[JSON] NewUserPublic
         :> MultiVerb 'POST '[JSON] RegisterResponses (Either RegisterError RegisterSuccess)
     )
+    -- This endpoint can lead to the following events being sent:
+    -- UserDeleted event to contacts of deleted user
+    -- MemberLeave event to members for all conversations the user was in (via galley)
+    -- TODO: put delete here, too?
```

Comment:
What does this TODO mean?  I suspect it's no longer valid; if you can't explain it either, we should probably remove it.